graylog2-server/src/main/java/org/graylog2/entitygroups/rest/EntityGroupPermissions.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.entitygroups.rest;
+
+public class EntityGroupPermissions {
+    public static final String ENTITY_GROUP_READ = "category:read";
+    public static final String ENTITY_GROUP_EDIT = "category:edit";
```

Comment:
Would we want to use the `entity_group` name in the permissions and audit names for consistency, or were we planning to refer to them as categories in the frontend?

Reply:
Not sure if we need to go there now, but it might be helpful to have a `type` enum for entity groups, so that they could be used for specific use cases. For example with Categories, we could assign a `CATEGORY` type, as a sort of namespace for all the security-specific categories. Not sure it's worth the effort at this point, but might be useful to nicely solve the naming difference.

Reply:
Makes sense to me, definitely good to wait until we have a use case.